input/intro-notes/StructureDefinition-au-core-patient-intro.md

```diff
@@ -48,4 +48,21 @@ By making Individual Gender Identity a Must Support data element, this profile e
   - Where the workflow does not support obtaining a gender identity value, it may be represented by sending the code "not-asked"
 - When exchanging concepts of sex or gender, refer to the guidance in the [Gender Harmony Implementation Guide](http://hl7.org/xprod/ig/uv/gender-harmony/) and [AU Base Patient](https://build.fhir.org/ig/hl7au/au-fhir-base//StructureDefinition-au-patient.html)
 
-
+<div class="request-for-feedback"><p>Specific feedback is sought on adding Must Support to specific parts of the HumanName in the AU Core Patient profile. Following the TDG agreement on making Patient.name.use a Must Support element, this proposal extends Must Support to:<ul>
```

Comment:
Too many words, shorten. These proposals will change over time in the JIRA ticket.